phase-6/submissions/create-relationship/CNazar

```diff
@@ -0,0 +1,2 @@
+D5045A696890C810C9E11B3D1DDD39BFB5381B9B2C028CAC02D002946920270E
+D5045A696890C810C9E11B3D1DDD39BFB5381B9B2C028CAC02D002946920270E
```

Comment:
Please, delete this file since it contains the wrong transaction hashes. 

You can delete it by executing: 
```
rm phase-6/submissions/create-relationship/CNazar
git add .
git commit -m "Removed wrong tx hash file" 
git push
```

Reply:
Done, pls check.Thx